managementApp/src/main/java/com/skytouch/controller/ManagmentController.java

```diff
@@ -0,0 +1,37 @@
+package com.skytouch.controller;
+
+import com.skytouch.service.ProductMessageSender;
+import com.skytouch.model.Product;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+
+@RestController
```

Comment:
Why are you using `@RestController` here?

Reply:
i use RestController only for test, these have already been changed to @controller